Firmware/mmu.cpp

```diff
@@ -1243,41 +1256,42 @@ void mmu_eject_filament(uint8_t filament, bool recover)
 
 void mmu_continue_loading() 
 {
-#ifdef MMU_IDLER_SENSOR_PIN
-			  for (uint8_t i = 0; i < MMU_IDLER_SENSOR_ATTEMPTS_NR; i++) {
 
-				  if (PIN_GET(MMU_IDLER_SENSOR_PIN) == 0) return;	
+	if (mmu_idler_sensor_detected) {
+		for (uint8_t i = 0; i < MMU_IDLER_SENSOR_ATTEMPTS_NR; i++) {
```

Comment:
There is danger in using macro instead of typed constant, that someone specify this macro larger than uin8_t MAX leading to infinite loop.